deployment/src/main/resources/templates/AsyncAPILoader.qute

```diff
@@ -0,0 +1,23 @@
+package {packageName};
+
+import io.quarkiverse.asyncapi.config.ObjectMapperFactory;
+import io.quarkiverse.asyncapi.config.AsyncAPIExtension;
+import io.quarkiverse.asyncapi.config.AsyncAPILoader;
+import com.asyncapi.v2.model.AsyncAPI;
+import java.io.IOException;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import javax.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped
+public class {id}AsyncAPILoader implements AsyncAPILoader {
+    public void load (Map<String, AsyncAPI> asyncAPIs) {
+      try {
+      	asyncAPIs.put("{id}", ObjectMapperFactory.get(AsyncAPIExtension.json).readValue("{asyncAPI}",AsyncAPI.class));
```

Comment:
This is now dumping the json content, but we can create subclasses of AsyncAPI like this one
https://github.com/Pakisan/jasyncapi/tree/master/asyncapi-plugin/asyncapi-plugin-maven#asyncapi-class-extending
This is the plan for next PRs